packages/synapse-react-client/src/components/ForumSearch/ForumSearch.tsx

```diff
@@ -12,6 +12,23 @@ import NoSearchResults from '../../assets/icons/NoSearchResults'
 import IconSvg from '../IconSvg/IconSvg'
 import { displayToast } from '../ToastMessage/ToastMessage'
 
+const NoSearchResultComponent = () => {
```

Comment:
Moved this component definition outside of its parent to prevent unnecessary re-rendering. Otherwise, only changed `NoSearchResults` to a component with styling to match styling of the previous SVG -- 

main | feature
:---:|:---:
<img width="1350" alt="SWC-6703_forumSearch_main" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/22347e2c-5937-4d7c-bc6e-a7daf60e519a"> | <img width="1355" alt="SWC-6703_forumSearch_feature" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/b43c5786-1e34-449b-9574-52da6af03c71">